source/guide_writefreely.rst

```diff
@@ -0,0 +1,167 @@
+.. author:: this.ven <lab@thisven.tuttle.uberspace.de>
+
+.. tag:: blog
+.. tag:: lang-go
+.. tag:: web
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/writefreely.svg
+      :align: center
+
+###########
+Writefreely
+###########
+
+.. tag_list::
+
+WriteFreely_ is built around writing. There's no news feed, notifications, or unnecessary likes or claps to take you away from your train of thought.
+Reach outside your own site with federation via ActivityPub. WriteFreely lets anyone on Mastodon, Pleroma, or any ActivityPub-enabled service follow your blog, bookmark your posts, and share them with their followers.
+WriteFreely is written in Go (golang), so it runs anywhere and takes up very few resources.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+  
+License
+=======
+
+Writefreely is released under the `AGPL v3.0`_.
+
+Prerequisites
+=============
+
+Set up your URL:
+
+.. include:: includes/web-domain-list.rst
+
+.. include:: includes/my-print-defaults.rst
+
+Installation
+============
+
+Use ``wget`` to download the `latest release`_:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget https://github.com/writeas/writefreely/releases/download/v0.12.0/writefreely_0.12.0_linux_amd64.tar.gz
+  [isabell@stardust ~]$ tar xzf writefreely_0.12.0_linux_amd64.tar.gz
+  [isabell@stardust ~]$
+
+Configuration
+=============
+
+Database Setup
+--------------
+
+Create a new database:
+
+.. code-block:: console
+
+  [isabell@stardust mailtrain]$ mysql -e "CREATE DATABASE ${USER}_writefreely CHARACTER SET latin1 COLLATE latin1_swedish_ci;"
+  [isabell@stardust mailtrain]$
+
+Writefreely Config
+------------------
+
+Change to _writefreely_ directory and start config mode to create a config file:
+
+.. note:: You need to select ``Production, behind reverse proxy`` in Server setup.
+
+.. code-block:: console
+
+  [isabell@stardust writefreely]$ cd ~/writefreely
+  [isabell@stardust writefreely]$ ./writefreely config start
+  [isabell@stardust writefreely]$
+  
+Alter _bind_ key and add TLS certificates paths in ``config.ini``; look for the ``[server]`` block:
+
+.. code-block:: ini
+
+  ...
+
+  bind                 = 0.0.0.0
+  tls_cert_path        = /home/isabell/etc/certificates/isabell.uber.space.crt
+  tls_key_path         = /home/isabell/etc/certificates/isabell.uber.space.key
```

Comment:
According to [the docs](https://writefreely.org/docs/latest/admin/config) a certificate is only needed when serving HTTPS directly using writefreely. On an uberspace HTTPS is handled by our nginx frontend, so there is no need to pass the certs here.

Reply:
Ok, i see.